portal/migrations/versions/40b223faa7d4_.py

```diff
@@ -0,0 +1,62 @@
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+
+
+"""empty message
+
+Revision ID: 40b223faa7d4
+Revises: 2126386257ad
+Create Date: 2018-01-18 14:58:36.140440
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '40b223faa7d4'
+down_revision = '2126386257ad'
+
+
+Session = sessionmaker()
+
+
+def update_job_name(session, old, new):
```

Comment:
@pbugni Is a migration for a scheduled job necessary? I thought any unmentioned ones will be deleted?

Reply:
@ivan-c Ivan, yeah, I chatted with Adam about this.  It seemed *unmentioned ones* were not deleted when I re-ran the jobs.  That is why I asked Adam to help me out with writing the migration script.

Reply:
Yea, I'm not sure where the disconnect here is, but both Amy and I tested this. Changing the names of the existing jobs in the ScheduledJob.json config file just leads to new jobs getting created with the new names, it doesn't remove the old ones.

We should definitely address that at some point so we don't need a migration for every job name change, but for now the migration should function as a work around.

Reply:
Thanks for investigating! I [created an issue,](https://jira.movember.com/browse/TN-592) let me know if I missed anything.